.prettierrc

```diff
@@ -0,0 +1,12 @@
+{
+    "printWidth": 120,
+    "semi": false,
+    "singleQuote": true,
+    "trailingComma": "all",
+    "bracketSpacing": false,
+    "jsxBracketSameLine": true,
+    "arrowParens": "avoid",
+    "insertPragma": true,
+    "tabWidth": 2,
+    "useTabs": false
+}
```

Comment:
❤️ that you added this! 💯

Reply:
Its meant to format my code... was meant to include it in git ignore.. thanks for pointing it out. still working on some components. will effect the changes